velox/vector/tests/utils/VectorMaker.h

```diff
@@ -54,6 +54,11 @@ class VectorMaker {
   static std::shared_ptr<const RowType> rowType(
       std::vector<std::shared_ptr<const Type>>&& types);
 
```

Comment:
add comment ` /// Create an RowVector from std::vector<std::vector<variant>>`
Should we follow `arrayOfRowVector()` implementation and make this a non-static function?

Reply:
Ok, fixed. Add `std::shared_ptr<velox::test::VectorMaker> vectorMaker_` in `ParquetTpchTestBase`, and make this function a non-static function.